Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -99,6 +102,7 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._appendStackTrace();
 
         this._renderRepeatCount();
+        this._renderTimestamp(true);            
```

Comment:
Style: When passing a constant literal value into a function, we prefer to declare a const member to name the variable for clarity.

e.g.
```
const renderLatestTimestamp = true;
this._renderTimestamp(renderLatestTimestamp);
```